spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/InternalServices.java

```diff
@@ -195,6 +204,13 @@ public Port managementServerPort() {
         return managementServerPort;
     }
 
+    /**
+     * Return true if {@link InternalServiceId#ACTUATOR} is specified in the properties of internal services.
+     */
+    public boolean actuatorEnabled() {
```

Comment:
We probably don't need this method anymore?
We can just check if `internalServicePort` is null or not in `ArmeriaSpringActuatorAutoConfiguration`.

Reply:
This method is used to get if `InternalServiceId#ACTUATOR` is specified in the property.
So removing this method means binding `WebOperationService` automatically into the internal service port if `internalServicePort` is not null and `armeria-spring-boot2-actuator-autoconfigure` is activated ?

Reply:
Oops, my bad. Yeah, we need this. Please forget about the comment. 😅